src/components_2/MyCard.js

```diff
@@ -1,31 +1,33 @@
 import * as React from 'react';
+
 import Card from '@mui/material/Card';
 import CardContent from '@mui/material/CardContent';
 import CardMedia from '@mui/material/CardMedia';
 import Typography from '@mui/material/Typography';
 import { CardActionArea } from '@mui/material';
-import Button from '@mui/material/Button';
 
-const MyCard = () => {
+
+const MyCard = ({name}, {stat}, {desc}) => {
```

Comment:
Hey, I am not sure if that is the correct syntax for restructuring. Try changing it to 

```
const MyCard = ({name, stat, desc}) => {
...
```